pandas/core/generic.py

```diff
@@ -3247,7 +3247,10 @@ def groupby(self, by=None, axis=0, level=None, as_index=True, sort=True,
             index. Only relevant for DataFrame input. as_index=False is
             effectively "SQL-style" grouped output
         sort : boolean, default True
-            Sort group keys. Get better performance by turning this off
+            Sort group keys. Get better performance by turning this off.
+            Note this does not influence the order of observations within each group. 
+            groupby preserves the sorted order of the target object
```

Comment:
sorted order -> order ? (as the order within a group is not necessarily sorted?)